app/Models/Chat/UserChannel.php

```diff
@@ -147,6 +152,7 @@ function ($userChannel) use ($byChannelId, $byUserId, $channelMessageIds, $userI
                     $userActual = $targetUser->userScoped;
                     $presence['icon'] = $userActual->user_avatar;
                     $presence['name'] = $userActual->username;
+                    $presence['moderated'] = !priv_check_user($user, 'ChatStart', $userActual)->can();
```

Comment:
should probably match this with sending messages which uses `ChatChannelSend`

Reply:
I think last I checked it is way more query heavy compared to this one

Reply:
it should probably include the `moderated` state still since that's part of the check when sending even if it's a pm channel